src/main/scala/com/fulcrumgenomics/umi/FilterConsensusReads.scala

```diff
@@ -56,11 +56,13 @@ import htsjdk.samtools.reference.ReferenceSequenceFileWalker
       |
       |  1. Reads with fewer than min-reads contributing reads are filtered out
       |  2. Reads with an average consensus error rate higher than max-read-error-rate are filtered out
-      |  3. Bases with quality scores below min-base-quality are masked to Ns
-      |  4. Bases with fewer than min-reads contributing raw reads are masked to Ns
-      |  5. Bases with a consensus error rate (defined as the fraction of contributing reads that
+      |  3. Reads with whose minimum depth less than min-depth are filtered out
+      |  4. Reads with mean base quality less than min-mean-base qulaity are filtered out (if specified)
```

Comment:
qulaity -> quality

Reply:
I know the ordering implies this, but I think we should explicitly state that this is the mean quality score of the consensus read _prior_ to any masking.